phpBB/language/en/install.php

```diff
@@ -300,11 +300,11 @@
 	'PHP_REGISTER_GLOBALS'			=> 'PHP setting <var>register_globals</var> is disabled',
 	'PHP_REGISTER_GLOBALS_EXPLAIN'	=> 'phpBB will still run if this setting is enabled, but if possible, it is recommended that register_globals is disabled on your PHP install for security reasons.',
 	'PHP_SAFE_MODE'					=> 'Safe mode',
-	'PHP_SETTINGS'					=> 'PHP version and settings',
-	'PHP_SETTINGS_EXPLAIN'			=> '<strong>Required</strong> - You must be running at least version 5.3.3 of PHP in order to install phpBB. If <var>safe mode</var> is displayed below your PHP installation is running in that mode. This will impose limitations on remote administration and similar features.',
+	'PHP_SETTINGS'					=> 'PHP versions and settings',
+	'PHP_SETTINGS_EXPLAIN'			=> '<strong>Required</strong> - You must be running at least versions 5.3.3 and less than 7 of PHP in order to install phpBB Ascraeus (3.1.x). If <var>safe mode</var> is displayed below your PHP installation is running in that mode. This will impose limitations on remote administration and similar features.',
```

Comment:
Still no good. Should be more like 
>  you  must be running at least version 5.3.3 of PHP (PHP 7 is not supported)) in order...,

And lose the as reads addition. It does not mean anything to end users and this change only applies to this version/branch of phpbb.